context.go

```diff
@@ -37,13 +38,16 @@ func NewContext(app *App, set *flag.FlagSet, parentCtx *Context) *Context {
 
 	if c.Context == nil {
 		ctx, cancel := context.WithCancel(context.Background())
+		signals := make(chan os.Signal, 1)
+		signal.Notify(signals, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
Because we capture TERM, perhaps we should not call it an interrupt handler but call it signal handler. I did not see that we capture more than a single signal before. Also, because we capture multiple, perhaps we should pass the signal to the handler.